test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -71,3 +71,41 @@ Import-Module Random
     }
 
 }
+
+Describe 'Module reloading with Class definition' -Tags "CI" {
+
+    BeforeAll {
+        Set-Content -Path TestDrive:\TestModule.psm1 -Value @'

```

Comment:
If this test loads the module from the file, it would not generate it every time and put into the asserts folder?